### DISCUSSION TITLE
New Release?

### DISCUSSION THREAD

**Discussion by linville** [atac/pychapter10#39] [General]
@mcferrill, any chance you have a new release scheduled at some point? I've been hot patching #31 and #33 where I use it and it'd be nice to be able to just use the regular release from PyPi.

Also, would you be interested in PRs to test/support newer versions of Python up to 5.12?

**Comment by mcferrill** [comment]
Ask and you shall receive! lmk if you run into any issues.

**Reply by mcferrill** [reply]
And I would certainly entertain PRs for more recent OS/python combinations! I attempted to add up to 3.11 to the pipeline while making the release, but azure choked on it for some reason and I don't have time to debug today.
